mobile/v1/redshift/sql-runner/sql/standard/04-sessions/01-main/02-sessions-sv-details.sql

```diff
@@ -0,0 +1,42 @@
+/*
+   Copyright 2020-2021 Snowplow Analytics Ltd. All rights reserved.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+DROP TABLE IF EXISTS {{.scratch_schema}}.mobile_sessions_screen_view_details{{.entropy}};
+
+CREATE TABLE {{.scratch_schema}}.mobile_sessions_screen_view_details{{.entropy}}
+  DISTSTYLE KEY
+  DISTKEY (session_id)
+  SORTKEY (session_id)
+AS (
+
+	SELECT
+		sv.session_id,
+		COUNT(DISTINCT sv.screen_view_id) AS screen_views,
+		COUNT(DISTINCT sv.screen_view_name) AS unique_screen_views,
```

Comment:
Not a fan of these names (as I think you note in a comment later on). 

I would intuitively understand `unique_screen_veiws` to be a distinct count of screen view ids. But this is actually a distinct count of screen names.

Possible alternatives to consider: `screens_viewed`, `screen_names_viewed`, `unique_screen_names`

Reply:
Good suggestions. I wasn't a huge fan of them either